terraform/queries/main.tf

```diff
@@ -0,0 +1,14 @@
+# Terraform configuration
+
+terraform {
+  required_providers {
+    honeycombio = {
+      source  = "honeycombio/honeycombio"
+      version = "~> 0.7.0"
```

Comment:
Inside a module you generally want to set a minimum and maximum version of the provider required: otherwise you have to update your module every release.


Consider ">= 0.7"?

Reply:
Oops! Yeah missed that bit. And makes sense - I'll refactor the board bit as well 👍

Reply:
Okay - I took this opportunity to add some more boards and queries 😅 The PR got a little bigger so apologies there! Happy to break it up if you prefer not to have this giant chunk in the git history.